bin/hdfcoinc/pycbc_multiifo_add_statmap

```diff
@@ -93,12 +112,27 @@ for f_in in files:
             all_trig_ids[ifo] = np.concatenate([all_trig_ids[ifo],
                                  -1 * np.ones_like(f_in['foreground/fap'][:],
                                                    dtype=np.uint32)])
+n_triggers = f['foreground/ifar'].size
+all_combo_types = []
```

Comment:
is `all_combo_types` used for anything?

Reply:
This was a hangover from the version which did not use the mask